sdk/iot/Azure.Iot.Hub.Service/src/Authentication/SharedAccessSignatureBuilder.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Azure.Iot.Hub.Service.Authentication
+{
+    internal sealed class SharedAccessSignatureBuilder
```

Comment:
why sealed?

Reply:
We don't want anyone inheriting from the sas token builder class, and overriding the buildToken or Sign implementations. Do you feel otherwise?

Reply:
It is internal so I was not sure if this is necessary. I don't see any harm though.

Reply:
Either we consistently use sealed, or we don't. Even for internal.

Reply:
So are we going to keep it as sealed or remove it?

Reply:
It is no longer a sealed class.